src/main/java/org/spongepowered/api/event/world/ExplosionEvent.java

```diff
@@ -36,16 +36,33 @@
 public interface ExplosionEvent extends TargetWorldEvent {
 
     /**
-     * Gets the {@link Explosion} involved in this event.
+     * An event that is fired before an explosion starts to occur.
      *
-     * @return The explosion that this event is involved in
+     * <p>The {@link Explosion} has not been created at this point.</p>
      */
-    Explosion getExplosion();
+    interface Pre extends ExplosionEvent, Cancellable {
+    }
 
     /**
-     * An event that is fired before the explosion occurs.
+     * An event that is fired when the explosion starts to occur.
+     *
+     * <p>The {@link Explosion} is available from here onwards.</p>
      */
-    interface Pre extends ExplosionEvent, Cancellable {
+    interface Start extends ExplosionEvent, Cancellable {
+
+        /**
+         * Gets the original {@link Explosion} involved in this event.
+         *
+         * @return The original explosion that this event is involved in
+         */
+        Explosion getOriginalExplosion();
```

Comment:
I can't make this a `Transaction<Explosion>` without finding a way to set the `explosion` field on the Forge `ExplosionEvent`.